x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/TransformIndexer.java

```diff
@@ -1155,25 +1155,31 @@ private SearchRequest buildQueryToUpdateDestinationIndex() {
 
     /**
      * Indicates if an audit message should be written when onFinish is called for the given checkpoint
-     * We audit the first checkpoint, and then every 10 checkpoints until completedCheckpoint == 99
-     * Then we audit every 100, until completedCheckpoint == 999
+     * We audit every checkpoint for the first 10 checkpoints until completedCheckpoint = 9.
+     * Then we audit every 10 checkpoints until completedCheckpoint == 99.
```

Comment:
Sorry, I'm not a native speaker so you will be better to judge here:
Could it be written "every 10th checkpoint", "every 100th checkpoint" etc.?

Reply:
Yes, that does sound a bit clearer. "Then we audit every 10th checkpoint until completedCheckpoint == 99."